### PR TITLE
add:

### PR DESCRIPTION
**Pull request by Varu98**
1) homepage
2) products
3) hero banner
4) Navbar

### CODE REVIEW

**Comment by Varu98** [comment]
recording


<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Chrome 97.0.4692.71 on Linux x86_64
Viewport:  1920 x 943 @1x
Language:  en-GB
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-1--delicate-truffle-d6ece4.netlify.app%2F&os=OS%20X&os_version=Big%20Sur&browser=Chrome&browser_version=97.0&resolution=1920x943&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-1--delicate-truffle-d6ece4.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=624befb022a59d65eacefd18&commentID=624bf2a6a9510459e76c9cea&resolution=resolved)

**Review by percius47 — APPROVED** [review]
The folder structure for images could have been pulled as links directly from one of the sites (eg. flipkart) for images like phones other than that some static banner images under assets holds good. Product images as direct links woulda have made the folder a bit lighter i guess. Rest looks good. Great job!

**Comment by Varu98** [comment]
> The folder structure for images could have been pulled as links directly from one of the sites (eg. flipkart) for images like phones other than that some static banner images under assets holds good. Product images as direct links woulda have made the folder a bit lighter i guess. Rest looks good. Great job!

Aah yes!! Thanks for the suggestion.

File: src/frontend/components/Home/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import "./Home.css";
+import { products } from "../../../backend/db/products";
+import ProductCard from "../ProductCard/ProductCard";
+
+const Home = () => {
+  return (
+    <div className="home-wrapper">
+      <div class="category-img-container">
+        <img class="category-img" src="assets/trending.jpg" alt="" />
+        <img class="category-img" src="assets/bestsellers.jpg" alt="" />
```
Comment (mimansha-swarup) [inline src/frontend/components/Home/Home.jsx]: change class to className

File: src/frontend/components/Home/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <img class="category-img" src="assets/latest.jpg" alt="" />
+        <img class="category-img" src="assets/valentine's.jpg" alt="" />
```
Comment (mimansha-swarup) [inline src/frontend/components/Home/Home.jsx]: don't keep alt empty

**Review by mimansha-swarup — APPROVED** [review]
just change class to className

File: src/frontend/components/Home/Home.jsx (hunk 1 above)
Comment (Varu98) [reply src/frontend/components/Home/Home.jsx]: Thanks Mimansha !!

**Pull request merged**